Packs/CloudConvert/Integrations/CloudConvert/CloudConvert.py

```diff
@@ -0,0 +1,377 @@
+
+        :return: dict containing the results of the export action as returned from the API (status, file ID, etc.)
+                    if the action was complete, the result url will be a part of this dict. If the request is pending,
+                    one should retrieve the url via the 'check_status' command
+        :rtype: ``Dict[str, Any]``
+        """
+        arguments['input'] = arguments.pop('task_id')
+        return self._http_request(
+            method='POST',
+            url_suffix='/export/url',
+            headers=self._headers,
+            data=arguments,
+            timeout=100,
+            ok_codes=(422, 200, 201, 500)
+        )
+    @logger
+    def get_file_from_url(self, url: str):
+        """
+        Call a GET http request in order to get the file data given as url
+        :param url: url containing a file
+        :return: request response, containing the data of the file
+        """
+        return self._http_request(
+            method='GET',
+            url_suffix=None,
+            full_url=url,
+            timeout=100,
+            headers={'Content-Type': 'application/json'},
+            resp_type='text'
+        )
+
+
+@logger
+def import_command(client: Client, arguments: Dict[str, Any]):
+    """
+    Import a file to the API for later conversion
+    :param client: CloudConvert client to use
+    :param arguments: All command arguments - either 'url' or 'entry_id'.
+    :return: CommandResults object containing the results of the import action as returned from the API and its
+             readable output
+    """
+
+    if arguments.get('url'):
+        results = client.import_url(arguments)
+        results_data = results.get('data')
+    elif arguments.get('entry_id'):
+        demisto.debug('getting the path of the file from its entry id')
+        result = demisto.getFilePath(arguments.get('entry_id'))
+        if not result:
+            raise ValueError('No file was found for given entry id')
+        file_path, file_name = result['path'], result['name']
+        results = client.import_entry_id(file_path, file_name)
+        results_data = results.get('data')
+    else:
+        raise ValueError('No url or entry id specified')
+
+    # No 'data' field was returned from the request, meaning the input was invalid
+    if results_data is None:
+        if results.get('message'):
+            raise ValueError(results.get('message'))
+        else:
+            raise ValueError(
+                'No response from server, check your request')
+
+    readable_output = tableToMarkdown('Import Results', remove_empty_elements(results_data),
+                                      headers=('created_at', 'id', 'operation', 'status'),
+                                      headerTransform=string_to_table_header)
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='CloudConvert.Task',
```

Comment:
So after discussing that with Lichter we realised this is a problem since the check-status command also updates the context depending on the operation. for example, if we did an import command, the context is updated and the 'status' field of that operation is (for example) 'waiting'. after that, a check-status command is called, and the results of this command will have, for that same operation, the status 'finished'. it makes more sense to keep this information under CloudConvert.Task.Import, as there is no point in keeping CloudConvert.Task.CheckStatus for all check status operations. this is true also for convert and export operations (meaning check-status should updates their contexts too). as a result of all these we assumed the better way about it is to keep the general CloudConver.Task, and keep the 'operation' field. what do you think?